pkg/reconciler/taskrun/taskrun_test.go

```diff
@@ -2545,8 +2590,14 @@ spec:
 		Kind:     "Task",
 		TaskSpec: &v1beta1.TaskSpec{Steps: simpleTask.Spec.Steps, Workspaces: simpleTask.Spec.Workspaces},
 	}
-	taskSpec := updateTaskSpecParamsContextsResults(context.Background(), taskRun, rtr)
-	pod, err := r.createPod(testAssets.Ctx, taskSpec, taskRun, rtr)
+	ctx := config.EnableAlphaAPIFields(context.Background())
```

Comment:
would you mind creating an issue to refactor these tests to call `Reconcile` rather than all of these helpers?

Reply:
Sure. I can do that.